src/Symfony/Component/Form/Extension/Core/Type/TimeType.php

```diff
@@ -62,6 +63,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         if ('single_text' === $options['widget']) {
             $builder->addEventListener(FormEvents::PRE_SUBMIT, static function (FormEvent $e) use ($options) {
+                /** @var PreSubmitEvent $event */
```

Comment:
In this method, `$event` is being used as argument of anonymous functions.
Shouldn't we update the type declaration for these arguments instead?

Reply:
See https://github.com/symfony/symfony/pull/52861#issuecomment-1836584376